packages/tiny-router/src/actions.ts

```diff
@@ -43,16 +44,24 @@ export const init: TinyRouter["actions"]["router"]["init"] = ({
   }
 };
 
-export const beforeSSR: TinyRouter["actions"]["router"]["beforeSSR"] = async ({
+export const beforeSSR: TinyRouter["actions"]["router"]["beforeSSR"] = ({
   state,
   actions
-}) => {
+}) => async (
+  ctx: Koa.ParameterizedContext<Koa.DefaultState, Koa.DefaultContext>
```

Comment:
For TypeScript I think I would expose a `ServerAction` in `"@frontity/types"` with our own `Context` type because it will include Koa's context but also more things in the future (like `ctx.settings`, `ctx.template` and so on...).

```ts
import { ServerAction } from "frontity/types";

actions: {
  myPackage: {
    beforeSSR: ServerAction<MyPackage>;
  ...
}
```

That way, people can launch other server actions from the `beforeSSR` one and organize their code better:

```ts
actions: {
  myPackage: {
    beforeSSR: ServerAction<MyPackage>;
    doSomethingWithContext: ServerAction<MyPackage>;
    doSomethingElseWithContext: ServerAction<MyPackage>;
  ...
}
```
```js
actions: {
  myPackage: {
    beforeSSR: ({ actions }) => async ({ ctx }) => {
      await Promise.all([
        actions.myPackage.doSomethingWithContext({ ctx }),
        actions.myPackage.doSomethingElseWithContext({ ctx })
      ]);
    }
  }
}
```